lib/libyui_client/http/libyui_http_client.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Http
+    class LibyuiHttpClient
+      def initialize(app)
+        @app = app
+      end
+
+      def widget_get(filter, timeout: LibyuiClient.timeout, interval: LibyuiClient.interval)
+        res = nil
+        Wait.until(timeout: timeout, interval: interval) do
+          uri = HttpClient.compose_uri(@app.host, @app.port, '/widgets', filter)
+          res = HttpClient.http_get(uri)
+          Response.new(res.code, res.body) if res.code.to_i == 200
```

Comment:
We could use something like `response.kind_of? Net::HTTPSuccess` here instead of hardcoded codes.